api-participant/src/main/java/bio/terra/pearl/api/participant/controller/PortalController.java

```diff
@@ -46,4 +54,34 @@ public ResponseEntity<Object> listPortalEnvAlerts(String portalShortcode, String
     return ResponseEntity.ok(
         portalDashboardConfigService.findByPortalEnvId(portalEnvironment.getId()));
   }
+
+  @Override
+  @CrossOrigin(
+      origins = {"https://juniperdemodev.b2clogin.com", "https://junipercmidemo.b2clogin.com"},
```

Comment:
What I actually want to do is populate this via `B2CConfigurationService.getOrigins`, but this annotation requires that `origins` is a constant. From Googling around, I think I'll need to make a custom cors filter for this